File: test-point-api/src/main/java/com/zrq/test/point/TestEntryPointInfo.java
package com.zrq.test.point;

/**
 * 描述：
 *
 * @author zhangrq
 * createTime 2020/12/7 14:23
 */
public class TestEntryPointInfo {
    int type;// 类型，1：Activity、2：Fragment、3：Support Fragment、4：静态无参方法（全部）、5：非静态无参方法（TestListActivity子类）
    String name;//
    String className;
    String methodName;

    public TestEntryPointInfo(int type, String name, String className, String methodName) {
        this.type = type;
        this.name = name;
        this.className = className;
        this.methodName = methodName;
    }
}

File: test-point-ksp/src/main/java/com/zrq/test/point/compiler/common/TestEntryPointInfo.kt
package com.zrq.test.point.compiler.common

/**
 * 描述：
 *
 * @author zhangrq
 * createTime 2024/4/15 下午3:36
 */
data class TestEntryPointInfo(
    val type: Int,
    val name: String,
    val className: String,
    val methodName: String?,
)
